src/pages/DatasetStatistics.tsx

```diff
@@ -43,12 +60,12 @@ export default function DatasetStatistics(props) {
     });
   }, [datasetIdentifier]);
 
-  const extract = useCallback((propertyName) => {
-    const property = find({propertyName})(dataset.properties);
+  const extract = useCallback((propertyName: string) => {
+    const property = find({propertyName})(dataset?.properties) as DatasetProperty;
```

Comment:
Would it be too ambitious to add types to the Ajax APIs too? If there's a way to incrementally add types to just the APIs used here that would be an improvement.

Reply:
Ajax often has types, but extract doesn't because properties is ambiguous.